src/main/java/com/eternalcode/parcellockers/ParcelLockers.java

```diff
@@ -74,29 +81,41 @@ public void onEnable() {
 
         HikariDataSource dataSource = DataSourceFactory.buildHikariDataSource(config, this.getDataFolder());
 
-        ParcelLockerRepository parcelLockerRepository = new ParcelLockerDatabaseService(dataSource);
+        ParcelLockerDatabaseService parcelLockerDatabaseService = new ParcelLockerDatabaseService(dataSource);
         ParcelDatabaseService parcelRepository = new ParcelDatabaseService(dataSource);
 
-        ParcelManager parcelManager = new ParcelManager(this, config, announcer, parcelRepository, parcelLockerRepository);
-        ParcelLockerManager parcelLockerManager = new ParcelLockerManager(parcelLockerRepository);
-
-        MainGUI mainGUI = new MainGUI(this, this.getServer(), miniMessage, config, parcelRepository, parcelLockerRepository);
-        ParcelListGUI parcelListGUI = new ParcelListGUI(this, this.getServer(), miniMessage, config, parcelRepository, parcelLockerRepository, mainGUI);
+        ParcelManager parcelManager = new ParcelManager(this, config, announcer, parcelRepository, parcelLockerDatabaseService);
+        ParcelLockerManager parcelLockerManager = new ParcelLockerManager(parcelLockerDatabaseService);
 
+        MainGUI mainGUI = new MainGUI(this, this.getServer(), miniMessage, config, parcelRepository, parcelLockerDatabaseService);
```

Comment:
Seperate `getServer()` method to the variable

Reply:
What is the advantage of using this? Or is it just a good practice?